GPU/Common/FramebufferManagerCommon.cpp

```diff
@@ -461,6 +469,26 @@ void FramebufferManagerCommon::DestroyFramebuf(VirtualFramebuffer *v) {
 	delete v;
 }
 
+void FramebufferManagerCommon::BlitFramebufferDepth(VirtualFramebuffer *src, VirtualFramebuffer *dst) {
+	bool matchingDepthBuffer = src->z_address == dst->z_address && src->z_stride != 0 && dst->z_stride != 0;
+	bool matchingSize = src->width == dst->width && src->height == dst->height;
+
+	// Note: we don't use CopyFramebufferImage here, because it would copy depth AND stencil.  See #9740.
+	if (matchingDepthBuffer && matchingSize) {
+		int w = std::min(src->renderWidth, dst->renderWidth);
+		int h = std::min(src->renderHeight, dst->renderHeight);
+		// Let's only do this if not clearing depth.
+		if (gstate_c.Supports(GPU_SUPPORTS_FRAMEBUFFER_BLIT)) {
+			draw_->BlitFramebuffer(src->fbo, 0, 0, w, h, dst->fbo, 0, 0, w, h, Draw::FB_DEPTH_BIT, Draw::FB_BLIT_NEAREST, "BlitFramebufferDepth");
+			RebindFramebuffer("BlitFramebufferDepth");
+		} else if (gstate_c.Supports(GPU_SUPPORTS_COPY_IMAGE)) {
+			draw_->CopyFramebufferImage(src->fbo, 0, 0, 0, 0, dst->fbo, 0, 0, 0, 0, w, h, 1, Draw::FB_DEPTH_BIT, "BlitFramebufferDepth");
+			RebindFramebuffer("BlitFramebufferDepth");
+		}
```

Comment:
does `BlitFramebuffer` here have higher priority than `CopyFramebufferImage` on purpose ?
`CopyFramebufferImage` should be faster no ?

Reply:
nvm just saw the comment above :).

Reply:
Actually not sure if that comment applies to all the backends, I think it got copy pasted from the GL backend..